Using GradleInspector with gradle.properties in ORT docker image

### What is the existing functionality and how should it be enhanced?


add 

# Ensure that the ORT data directory exists to be able to mount the config into it with correct permissions.
RUN mkdir -p "$HOME/.ort" **"$HOME/.gradle"**

in line 598 of Dockerfile


### What is the use-case for your enhancement?

I want to use the existing ort docker image from ghcr.io/oss-review-toolkit/ort with gradle / gradleInspector package manager using an own gradle.properties file  

Mounting the gradle.properties file during docker run with -v ~/gradle.properties:/home/ort/.gradle/gradle.properties will make ROOT the owner /home/ort/.gradle  (e.g. using DOCKER DESKTOP / DOCKER CE on Windows)


### Alternatives you have considered

Docker CE on Linux works fine, Docker on Windows with gradle is not working when gradle.properites must be mounted to container.

### ERROR when using the current ort image

10:27:12.211 [DefaultDispatcher-worker-1] ERROR org.ossreviewtoolkit.analyzer.PackageManager - GradleInspector failed to resolve dependencies for path 'settings.gradle': GradleConnectionException: Could not install Gradle distribution from '[https://services.gradle.org/distributions/gradle-8.10.2-bin.zip'.](https://services.gradle.org/distributions/gradle-8.10.2-bin.zip%27.)
Caused by: RuntimeException: Could not create parent directory for lock file /home/ort/.gradle/wrapper/dists/gradle-8.10.2-bin/a04bxjujx95o3nb99gddekhwo/gradle-8.10.2-bin.zip.lck

--- a/Dockerfile
+++ b/Dockerfile
@@ -582,8 +582,8 @@ ENV PATH=$PATH:/opt/ort/bin
 USER $USER
 WORKDIR $HOME
 
-# Ensure that the ORT data directory exists to be able to mount the config into it with correct permissions.
-RUN mkdir -p "$HOME/.ort"
+# Ensure that these directories exist in the container to be able to mount directories from the host into them with correct permissions.
+RUN mkdir -p "$HOME/.ort" "$HOME/.gradle"
 
 ENTRYPOINT ["/opt/ort/bin/ort"]
 
@@ -598,7 +598,7 @@ ENV PATH=$PATH:/opt/ort/bin
 USER $USER
 WORKDIR $HOME
 
-# Ensure that the ORT data directory exists to be able to mount the config into it with correct permissions.
-RUN mkdir -p "$HOME/.ort"
+# Ensure that these directories exist in the container to be able to mount directories from the host into them with correct permissions.
+RUN mkdir -p "$HOME/.ort" "$HOME/.gradle"
 
 ENTRYPOINT ["/opt/ort/bin/ort"]

